scripts/gatsby-plugin-checker/README.md

```diff
@@ -1,3 +1,5 @@
 # gatsby-plugin-checker
 
 This script searches the npm API for plugins that start with either "gatsby-source", "gatsby-plugin", or "gatsby-transformer" but don't have the "gatsby-plugin" keyword in their `package.json` and thus are not included in Gatsby's keyword search. The script will then notify those repositories by creating an issue notifying them to add the keyword to their `package.json` (This functionality will be included in the next commit). Once a repo has been notified, its `notified` attribute will be changed to `true` in `plugins.json` and won't be notified. Similarly, if a repo is a false positive or doesn't want to be notified, it can be `blacklisted`.
+
+To run this script you have to create a `.env` file in this directory and create there a variable `GITHUB_PERSONAL_ACCESS_TOKEN` and assign there your personal access token. You can find an instruction for generating it [here](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line). You shouldn't commit `.env` file.
```

Comment:
Nit: but could we name this `GITHUB_API_TOKEN`? It's been used in www/ and elsewhere with that name, so it might help ease confusion for any who happen to contribute to multiple areas of the codebase.

```suggestion
To run this script you have to create a `.env` file in this directory and create there a variable `GITHUB_API_TOKEN` and assign there your personal access token. You can find an instruction for generating it [here](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line). You shouldn't commit `.env` file.
```